DOC: Updated links to 2 tutorials in tutorials.rst

tommyod opened the pull request:
Two of the links to the [pandas-cookbook](https://github.com/jvns/pandas-cookbook) did not work for version `v0.2` of the documents. Now all links work. Where a version  `v0.2` exists, it links to that version. In the remaining two cases, it now links to version `v0.1` instead of raising the `404: Page not found` error.

tommyod committed:
Changed links to 2 tutorials from v0.2 to v0.1, as v0.2 did not exist for all of them

jorisvandenbossche commented:
I would rather update the actual links and keep linking to the newer version, as those chapters still exist (the name was a bit changed): http://nbviewer.jupyter.org/github/jvns/pandas-cookbook/tree/master/cookbook/

tommyod committed:
Updated all pandas-tutorial links to v0.2, added chapter 9 of tutorial

tommyod commented:
I agree @jorisvandenbossche. Thanks for reviewing! I have updated this PR:

- Every link is to `v2.0` now.
- Another chapter has been written (Chapter 9), I have added it.

And I believe this PR is good to go then.

jreback commented:
thanks!
